buildSrc/src/main/kotlin/dokka-conventions.gradle.kts

```
@@ -18,64 +20,58 @@ configure(subprojects.filterNot { projetsWithoutDokka.contains(it.name) }) {
     configureExternalLinks()
 }
 
-// Setup top-level 'dokkaHtmlMultiModule' with templates
-tasks.withType<DokkaMultiModuleTask>().named("dokkaHtmlMultiModule") {
-    setupDokkaTemplatesDir(this)
+// For top-level multimodule collection
+dokka {
+    setupDokkaTemplatesDir()
 }
-
 dependencies {
-    // Add explicit dependency between Dokka and Knit plugin
-    add("dokkaHtmlMultiModulePlugin", "org.jetbrains.kotlinx:dokka-pathsaver-plugin:$knit_version")
+    dokkaPlugin("org.jetbrains.kotlinx:dokka-pathsaver-plugin:$knit_version")
+
+    subprojects.forEach {
+        if (it.name !in projetsWithoutDokka) {
```

Minor typo.

```suggestion
        if (it.name !in projectsWithoutDokka) {
```

(I made a suggestion to make it easier to see the typo, but maybe other places in the code should be updated.)